CHANGELOG.md

```diff
@@ -1,5 +1,9 @@
 # Changelog
 
-## Unreleased 
+## Unreleased
 
 - feat: Add healthchecks for redis, memcached and postgres (#975)
+
+### Breaking
+- The frontend bundle will be loaded asynchronously. This is a breaking change that can affect custom plugins that access certain globals in the django template. Please see https://forum.sentry.io/t/breaking-frontend-changes-for-custom-plugins/14184 for more information.
```

Comment:
Let's align this with the commit log format we are already following<sup>[1](https://github.com/angular/angular/blob/master/CONTRIBUTING.md#-commit-message-format),[2](https://github.com/semantic-release/semantic-release)</sup>:

```suggestion
- BREAKING CHANGE: The frontend bundle will be loaded asynchronously. This is a breaking change that can affect custom plugins that access certain globals in the django template. Please see https://forum.sentry.io/t/breaking-frontend-changes-for-custom-plugins/14184 for more information.
```